network-store-server/src/main/java/com/powsybl/network/store/server/Mappings.java

```diff
@@ -389,10 +431,31 @@ private void createDanglingLineMappings() {
         danglingLineMappings.addColumnMapping(ALIASES_WITHOUT_TYPE, new Mapping<>(Set.class, DanglingLineAttributes::getAliasesWithoutType, DanglingLineAttributes::setAliasesWithoutType));
         danglingLineMappings.addColumnMapping("generation", new Mapping<>(DanglingLineGenerationAttributes.class, DanglingLineAttributes::getGeneration, DanglingLineAttributes::setGeneration));
         danglingLineMappings.addColumnMapping("ucteXnodeCode", new Mapping<>(String.class, DanglingLineAttributes::getUcteXnodeCode, DanglingLineAttributes::setUcteXnodeCode));
-        danglingLineMappings.addColumnMapping("currentLimits", new Mapping<>(LimitsAttributes.class, DanglingLineAttributes::getCurrentLimits, DanglingLineAttributes::setCurrentLimits));
         danglingLineMappings.addColumnMapping(POSITION, new Mapping<>(ConnectablePositionAttributes.class, DanglingLineAttributes::getPosition, DanglingLineAttributes::setPosition));
-        danglingLineMappings.addColumnMapping("apparentPowerLimits", new Mapping<>(LimitsAttributes.class, DanglingLineAttributes::getApparentPowerLimits, DanglingLineAttributes::setApparentPowerLimits));
-        danglingLineMappings.addColumnMapping("activePowerLimits", new Mapping<>(LimitsAttributes.class, DanglingLineAttributes::getActivePowerLimits, DanglingLineAttributes::setActivePowerLimits));
+        danglingLineMappings.addColumnMapping("permanentCurrentLimit", new Mapping<>(Double.class,
```

Comment:
Add a `private static final String` for `"permanentCurrentLimit"` ?

Reply:
Done